hermes-management/src/main/java/pl/allegro/tech/hermes/management/domain/Auditor.java

```diff
@@ -1,13 +1,13 @@
 package pl.allegro.tech.hermes.management.domain;
 
-public interface Auditor {
-    default void objectCreated(String username, Object createdObject) {
+public interface Auditor<T extends Object> {
```

Comment:
this is now a generic interface, yet you lose generic information in all places where it is used - maybe we don't need it to be generic after all? I mean Auditor is here to audit, not to mingle with objects too much